src/storage/test/QueryVertexPropsTest.cpp

```diff
@@ -48,6 +49,8 @@ void mockData(kvstore::KVStore* kv,
                 baton.post();
             });
         baton.wait();
+        kvstore::ResultCode code = kv->flush(0);  // flush per partition
```

Comment:
What is the reason for flush here?

Reply:
We should generate sstable filte to test prefix bloom filter take effect. @panda-sheep